Dockerfiles/mods/Dockerfile-8.0

```diff
@@ -537,6 +537,17 @@ RUN set -eux \
 	&& true
 
 
+# -------------------- Installing PHP Extension: swoole --------------------
```

Comment:
This is unfortunately not how it works. Dockerfiles are generated via Ansible from the configuration here. See section with swoole:

https://github.com/devilbox/docker-php-fpm/blob/master/build/ansible/group_vars/all/mods.yml#L989